<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Armour Password Manager</string>
  </property>
  <widget class="QWidget" name="horizontal_table_layout">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,4">
    <item>
     <widget class="QListWidget" name="listWidget_groups">
      <item>
       <property name="text">
        <string>Test 1</string>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Test 2</string>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Test 3</string>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget_entries"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>888</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>Database</string>
    </property>
    <addaction name="actionNew_Database"/>
    <addaction name="actionOpen_Database"/>
    <addaction name="actionSave_Database"/>
    <addaction name="actionSave_Database_as"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Exit"/>
   </widget>
   <widget class="QMenu" name="menuEntry">
    <property name="title">
     <string>Entry</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuGroups">
    <property name="title">
     <string>Groups</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuDatabase"/>
   <addaction name="menuGroups"/>
   <addaction name="menuEntry"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Database"/>
  </widget>
  <action name="actionNew_Database">
   <property name="icon">
    <iconset resource="../src/resources.qrc">
     <normaloff>:/document/icons/Document/file-add-line.svg</normaloff>:/document/icons/Document/file-add-line.svg</iconset>
   </property>
   <property name="text">
    <string>New Database</string>
   </property>
  </action>
  <action name="actionOpen_Database">
   <property name="icon">
    <iconset resource="../src/resources.qrc">
     <normaloff>:/document/icons/Document/folder-open-line.svg</normaloff>:/document/icons/Document/folder-open-line.svg</iconset>
   </property>
   <property name="text">
    <string>Open Database</string>
   </property>
  </action>
  <action name="actionSave_Database">
   <property name="icon">
    <iconset resource="../src/resources.qrc">
     <normaloff>:/device/icons/Device/save-2-line.svg</normaloff>:/device/icons/Device/save-2-line.svg</iconset>
   </property>
   <property name="text">
    <string>Save Database</string>
   </property>
  </action>
  <action name="actionSave_Database_as">
   <property name="icon">
    <iconset resource="../src/resources.qrc">
     <normaloff>:/document/icons/Document/file-copy-2-line.svg</normaloff>:/document/icons/Document/file-copy-2-line.svg</iconset>
   </property>
   <property name="text">
    <string>Save Database as</string>
   </property>
  </action>
  <action name="actionSave_Exit">
   <property name="icon">
    <iconset resource="../src/resources.qrc">
     <normaloff>:/system/icons/System/close-circle-line.svg</normaloff>:/system/icons/System/close-circle-line.svg</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../src/resources.qrc">
     <normaloff>:/system/icons/System/question-line.svg</normaloff>:/system/icons/System/question-line.svg</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../src/resources.qrc"/>
 </resources>
 <connections/>
</ui>
